autoscaler/cluster.py

```diff
@@ -639,3 +648,7 @@ def maintain(self, cached_managed_nodes, running_insts_map,
                         else:
                             logger.info(
                                 '[Dry run] Would have terminated unmanaged %s', inst)
+
+    def _map_sets_to_values(self, set_map):
+        for key in set_map.keys():
+            set_map[key] = int(min(set_map[key]))
```

Comment:
`min` here is not about sorting. There should only ever be one item in each set. `min` just gets it out - using `max` here should have the same effect.